gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt

```diff
@@ -415,7 +417,11 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
                         configureAsAbstractKotlinCompileTool(kspTask as AbstractKotlinCompileTool<*>)
                         configurePluginOptions(kspTask)
                         kspTask.compilerOptions.noJdk.value(kotlinCompileTask.compilerOptions.noJdk)
-                        kspTask.compilerOptions.useK2.value(false)
+                        kspTask.compilerOptions.languageVersion.orNull?.let { version ->
```

Comment:
Configuration of shared compilerOptions can be moved into a local function:
```
fun configureKotlinCompilationTask(task: KotlinCompilationTask) {
  ...
}
```

Reply:
done